src/components/input/SelectNext.tsx

```diff
@@ -212,11 +217,11 @@ function SelectMain<T>({
       className={classnames('relative w-full border rounded', inputGroupStyles)}
       ref={wrapperRef}
     >
-      <Button
+      <button
         id={buttonId ?? defaultButtonId}
-        variant="custom"
-        classes={classnames(
-          'w-full flex justify-between',
+        className={classnames(
+          'focus-visible-ring transition-colors whitespace-nowrap',
+          'w-full flex items-center justify-between gap-x-2 p-2',
```

Comment:
These classes were automatically applied by our `Button` component, but we have moved to the regular `button` because our own only sets `aria-label` via `title`.